src/theme.css

```diff
@@ -1,43 +1,93 @@
-  --yellow-100: #ffd89f;
-  --yellow-200: #fef7d2;
-  --yellow-800: #ffb03f;
-  --fuchsia-200: #ffe4ff;
-  --fuchsia-800: #ff5afe;
-  --green-200: #e1f3d6;
-  --green-800: #44b600;
-  --green-900: #328500;
-  --red-100: rgba(216, 0, 39, 0.16);
-  --red-150: rgba(216, 0, 39, 0.3);
-  --red-200: #f9d6dc;
-  --red-800: #d80027;
-  --blue-200: #e2ecfc;
-  --blue-600: #89baff;
-  --blue-800: #4888ee;
-  --gray: #a19ea8; /* TODO: Sync with Figma color styles */
-  --white-900: #ffffff;
-  --orange-200: #ffe9df;
-  --orange-800: #ff7439;
-  --black-100: rgba(115, 110, 125, 0.16);
-  --black-200: #f3f2f5;
-  --black-300: #dddce0;
-  --black-400: #b9b7be;
-  --black-600: #6e6977;
-  --black-700: #37333d;
-  --black-800: #16141a;
-  --purple-200: #e0dcf3;
-  --purple-800: #3c24b3;
-  --dcl-primary: #ff2d55;
-
-  --background-catalyst: rgba(72, 136, 238, 0.08);
-  --background-poi: rgba(68, 182, 0, 0.08);
-  --background-ban-name: rgba(255, 90, 254, 0.08);
-  --background-grant: rgba(60, 36, 179, 0.08);
-  --background-poll: rgba(255, 116, 57, 0.08);
-  --background-remove-poi: #fff1f4;
-  --background-linked-wearables: #fef9f0;
-  --background-popup: #5c5c6e;
-  --background-disabled: #f9f8f9;
+  --yellow-raw: 255, 176, 63;
+  --fuchsia-raw: 255, 90, 254;
+  --green-raw: 68, 182, 0;
+  --green-raw-b: 1, 127, 10;
+  --red-raw: 216, 0, 39;
+  --red-raw-b: 255, 0, 46;
+  --red-raw-c: 216, 0, 39;
+  --red-raw-d: 255, 0, 0;
+  --blue-raw: 72, 136, 238;
+  --blue-raw-b: 0, 191, 217;
+  --blue-raw-c: 64, 95, 205;
+  --gray-raw: 161, 158, 168;
+  --gray-raw-b: 115, 110, 125;
+  --white-raw: 255, 255, 255;
+  --orange-raw: 255, 116, 57;
+  --black-raw: 22, 20, 26;
+  --purple-raw: 60, 36, 179;
+  --purple-raw-b: 198, 64, 205;
+  --purple-raw-c: 219, 0, 255;
+  --purple-raw-d: 132, 17, 144;
+  --purple-raw-e: 165, 36, 179;
+  --dcl-primary-raw: 255, 45, 85;
+
+  --gradient-green-fancy: linear-gradient(90deg, rgba(var(--green-raw-b), 1) 0%, rgba(var(--green-blue-b), 1) 100%);
+  --gradient-red-fancy-1: linear-gradient(90deg, rgba(var(--red-raw-b), 1) 0%, rgba(var(--purple-raw-b), 1) 100%);
+  --gradient-red-fancy-2: linear-gradient(90deg, rgba(var(--red-raw-c), 1) 0%, rgba(var(--red-raw-b), 1) 100%);
+  --gradient-purple-fancy-1: linear-gradient(90deg, rgba(var(--purple-raw-b), 1) 0%, rgba(var(--blue-raw-c), 1) 100%);
+  --gradient-purple-fancy-2: linear-gradient(90deg, rgba(var(--purple-raw-c), 1) 0%, rgba(var(--purple-raw-d), 1) 100%);
+  --gradient-purple-fancy-3: linear-gradient(90deg, rgba(var(--purple-raw-e), 1) 0%, rgba(var(--purple-raw-d), 1) 100%);
+
+  --yellow-100: rgba(255, 216, 159, 1);
+  --yellow-200: rgba(254, 247, 210, 1);
+  --yellow-800: rgba(var(--yellow-raw), 1);
+
+  --fuchsia-200: rgba(255, 228, 255, 1);
+  --fuchsia-200t: rgba(var(--fuchsia-raw), 0.16);
+  --fuchsia-500t: rgba(var(--fuchsia-raw), 0.5);
+  --fuchsia-800: rgba(var(--fuchsia-raw), 1);
+
+  --green-200: rgba(225, 243, 214, 1);
+  --green-200t: rgba(var(--green-raw), 0.16);
```

Comment:
What's the `t` in the theme colors? 🤔

Reply:
Visually it would be the same color - except one uses relies on Alpha values and the other is solid (100% opacity)

Reply:
This is bound to change as soon as we implement some upcoming CSS color goodies and/or CSS preprocessors that would enable us to operate with colors in a more programatic way. Think expressions/functions like darken/lighten/mix.

Reply:
> I don't like the abbreviation. Maybe `--transparent-green-200`... Why isn't it another color instead of a green-200 variation?

I also don't understand this, if we can have different shades of a color expressed as a numerical value: can the transparent ones be also a color between 0 and 999?

Reply:
Take the reds for example:

```css
  --red-100: rgba(var(--red-raw), 0.16);
  --red-200: rgba(249, 214, 220, 1);
  --red-200t: rgba(var(--red-raw), 0.16);
```

We have `red-200t` that is the same as `red-100`. We could just use `red-100` right?

And in the same way, we could do that for the greens, instead of 

```css
  --green-200: rgba(225, 243, 214, 1);
  --green-200t: rgba(var(--green-raw), 0.16);
```

we could have 

```css
  --green-100: rgba(var(--green-raw), 0.16);
  --green-200: rgba(225, 243, 214, 1);
  --green-400: rgba(var(--green-raw), 0.4);
  --green-800: rgba(var(--green-raw), 1);
  --green-900: rgba(50, 133, 0, 1);
  ```
  
WDYT?

Reply:
We can also think the colors with alpha as another color palette.
I find useful to check what color is being used in the figma by inspecting the component and finding the color style name. Are we going to keep that 1:1 relationship with figma?